OpenRA.Mods.Common/Effects/NukeLaunch.cs

```diff
@@ -36,12 +37,13 @@ public class NukeLaunch : IEffect
 		WPos pos;
 		int ticks;
 
-		public NukeLaunch(Player firedBy, string name, WeaponInfo weapon, string weaponPalette,
+		public NukeLaunch(Player firedBy, string name, WeaponInfo weapon, string weaponPalette, string up, string down,
```

Comment:
`upSequence` and `downSequence` please, to make it clear what these actually are.